Process.c

```diff
@@ -64,29 +64,26 @@ void Process_fillStarttimeBuffer(Process* this) {
  */
 #define TASK_COMM_LEN 16
 
-static bool findCommInCmdline(const char* comm, const char* cmdline, int cmdlineBasenameStart, int* pCommStart, int* pCommEnd) {
+static bool findCommInCmdline(const char* comm, const char* cmdline, size_t cmdlineBasenameStart, size_t* pCommStart, size_t* pCommLen) {
    /* Try to find procComm in tokenized cmdline - this might in rare cases
     * mis-identify a string or fail, if comm or cmdline had been unsuitably
     * modified by the process */
    const char* tokenBase;
    size_t tokenLen;
    const size_t commLen = strlen(comm);
 
-   if (cmdlineBasenameStart < 0)
-      return false;
```

Comment:
This removes a check when no basename was found in the command line …

Reply:
"No basename found" is no different from matching "comm" from the start of the "cmdline" string. You notice this function does not take `cmdlineBasenameLen` as input. "No basename found" may be indicated by `cmdlineBasenameLen == 0`, but it's irrelevant to this function.

Reply:
Just checking the change in the code has been accounted for in a different way.